compiler/rustc_monomorphize/src/collector.rs

```diff
@@ -796,7 +809,7 @@ impl<'a, 'tcx> MirVisitor<'tcx> for MirNeighborCollector<'a, 'tcx> {
                 collect_const_value(self.tcx, const_val, self.output)
             }
             ty::ConstKind::Unevaluated(unevaluated) => {
-                match self.tcx.const_eval_resolve(param_env, unevaluated, None) {
+                match self.tcx.const_eval_resolve(param_env, unevaluated.expand(), None) {
```

Comment:
`visit_const` should be removed :laughing: should already be covered by `visit_constant`